types/react-native/index.d.ts

```diff
@@ -469,6 +471,84 @@ export interface Insets {
     right?: number;
 }
 
+export type PressableStateCallbackType = Readonly<{
+    pressed: boolean;
+}
+
+export interface PressableProps extends AccessibilityProps, Omit<ViewProps, 'style' | 'hitSlop'> {
+    /**
+     * Called when a single tap gesture is detected.
+     */
+    onPress?: null | (event: GestureResponderEvent) => void;
```

Comment:
Missing parenthesis for all callbacks: `onPress?: null | ((event: GestureResponderEvent) => void);`